app/code/Magento/Paypal/Test/Unit/Model/Api/NvpTest.php

```diff
@@ -197,7 +197,7 @@ public function callDataProvider()
                 10417
             ],
             [
-                "\r\n" . 'ACK[7]=Failure&L_ERRORCODE0[5]=10417&L_SHORTMESSAGE0[8]=Message.',
+                "\r\n" . 'ACK[7]=Failure&L_ERRORCODE0[5]=10417&L_SHORTMESSAGE0[8]=Message.&L_LONGMESSAGE0[15]=',
```

Comment:
Why did it change?

Reply:
I assume that this item exist in dataProvider to cover case when `L_LONGMESSAGE` absent in response. Please adjust method instead of test